.github/workflows/multi_nim.yml

```diff
@@ -0,0 +1,157 @@
+name: Daily
+on:
+  schedule:
+    - cron: "30 5 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - os: linux
+            cpu: amd64
+          - os: linux
+            cpu: i386
+          - os: macos
+            cpu: amd64
+          - os: windows
+            cpu: amd64
+          #- os: windows
+            #cpu: i386
+        branch: [version-1-2, version-1-4, version-1-6, devel]
+        include:
+          - target:
+              os: linux
+            builder: ubuntu-18.04
+            shell: bash
+          - target:
+              os: macos
+            builder: macos-10.15
+            shell: bash
+          - target:
+              os: windows
+            builder: windows-2019
+            shell: msys2 {0}
+
+    defaults:
+      run:
+        shell: ${{ matrix.shell }}
+
+    name: '${{ matrix.target.os }}-${{ matrix.target.cpu }} (Nim ${{ matrix.branch }})'
+    runs-on: ${{ matrix.builder }}
+    continue-on-error: ${{ matrix.branch == 'version-1-6' || matrix.branch == 'devel' }}
+    steps:
+      - name: Checkout nim-chronos
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          path: nim-chronos
+          submodules: false
+
+      - name: Install build dependencies (Linux i386)
+        if: runner.os == 'Linux' && matrix.target.cpu == 'i386'
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt-fast update -qq
+          sudo DEBIAN_FRONTEND='noninteractive' apt-fast install \
+            --no-install-recommends -yq gcc-multilib g++-multilib \
+            libssl-dev:i386
+          mkdir -p external/bin
+          cat << EOF > external/bin/gcc
+          #!/bin/bash
+          exec $(which gcc) -m32 "\$@"
+          EOF
+          cat << EOF > external/bin/g++
+          #!/bin/bash
+          exec $(which g++) -m32 "\$@"
+          EOF
+          chmod 755 external/bin/gcc external/bin/g++
+          echo '${{ github.workspace }}/external/bin' >> $GITHUB_PATH
+
+      - name: MSYS2 (Windows i386)
+        if: runner.os == 'Windows' && matrix.target.cpu == 'i386'
+        uses: msys2/setup-msys2@v2
```

Comment:
Sorry but i still do not like this approach, because this setup is almost impossible to use not in CI. 
So i prefer old approach with different URLs, this distributions do not have any dependency and do not require you to setup `msys2` which is not compatible with `Git For Windows`.

Reply:
https://github.com/status-im/nim-chronos/blob/master/.github/workflows/ci.yml#L83-L94

Reply:
You might want to see how the sausage is made, first: https://github.com/brechtsanders/winlibs_mingw/issues/30

Reply:
I'm fine with it. And i really do not care how the sausage is made but i do not like sausages which could be eaten only with predefined plate and sauce. But looks like you like to eat sausages which will become paper if you change plate or sauce.